local_services/mds_data_generator/cleanup_test_data.py

```diff
@@ -0,0 +1,149 @@
+import os
+
+import boto3
+from botocore.config import Config
+from db_utils import get_db_connection
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def clean_database():
+    """Clean up all test data from the database."""
+    conn = get_db_connection()
+
```

Comment:
Instead of this, I wonder if the following utilising the SQLAlchemy ORM is sufficient

```
from app.main.db.models import db

db.drop_all()
db.create_all()
```